terraform/forward_email/preview.tfbackend

```diff
@@ -0,0 +1,4 @@
+bucket = "terraform-state-20240618152116874600000001"
```

Comment:
Sorry, I'm migrating to a new tfstate bucket. Please use this S3 bucket to store the tfstate files.
Ref: https://aws-educate-cloud-ambassador-dev-team.atlassian.net/browse/SCRUM-375
```suggestion
bucket = "tpet-terraform-state-20250401155336893000000001"
```